src/components/new-safe/steps/Step1/index.tsx

```diff
@@ -48,8 +49,13 @@ function CreateSafeStep1({
     },
   })
 
+  const onFormSubmit = (data: Partial<NewSafeFormData>) => {
+    setSafeName(data.name ?? '')
```

Comment:
We should fallback to the placeholder mnemonic, not an empty string.

Reply:
I'm solving this now. we were also not setting the data with the fallback